bin/cli.js

```diff
@@ -9,75 +10,96 @@ const yargs = require('yargs');
 const PWMetrics = require('../lib/index');
 const { getConfigFromFile } = require('../lib/utils/fs');
 const { getMessageWithPrefix } = require('../lib/utils/messages');
+const { mergeDeep } = require('../lib/utils/object');
 
 const cliFlags = yargs
   .help('help')
   .version(() => require('../package').version)
   .showHelpOnFail(false, 'Specify --help for available options')
-
-  .usage('$0 url')
-
-  .describe({
-    'json': 'Reports json details to stdout',
-    'runs': 'Does n runs (eg. 3, 5), and reports the median run\'s numbers.',
-    'expectations': 'Expectations from metrics results. Useful for CI. Config required.',
-    'submit': 'Submit metric results into sheets. Config required.',
-    'config': 'Read config form file.',
-    'disable-cpu-throttling': 'Disable CPU throttling',
-    'and more Lighthouse CLI options': 'Docs - https://github.com/GoogleChrome/lighthouse/#lighthouse-cli-options',
+  .wrap(yargs.terminalWidth())
+  .usage('Usage: $0 <url>')
+  .command('url', 'URL to test')
+  .option('json', {
```

Comment:
If we are going with output-path we can use output flag instead of json. It might have different values like stdout (default) and json (csv in future). It will add some flexibility.